src/registrar/templates/admin/fieldset.html

```diff
@@ -0,0 +1,72 @@
+{% load i18n static %}
+
+{% comment %}
+This is copied from Djangos implementation of this template, with added "blocks"
+It is not inherently customizable on its own, so we can modify this instead.
+https://github.com/django/django/blob/main/django/contrib/admin/templates/admin/includes/fieldset.html
+{% endcomment %}
+<fieldset class="module aligned {{ fieldset.classes }}">
+    {# .gov override #}
```

Comment:
Keep only the custom blocks you are using, remove the other forward looking ones. The benefits of having them may be outweighed by tech debt on django updates (potentially)